Coralogix/Sources/Utils/Helper.swift

```diff
@@ -11,6 +11,35 @@ import UIKit
 #endif
 
 class Helper {
+    internal static func convertToAnyDict(_ attributeDict: [String: AttributeValue]) -> [String: Any] {
+        var anyDict: [String: Any] = [:]
+
+        for (key, attributeValue) in attributeDict {
+            switch attributeValue {
+            case let .string(value):
+                anyDict[key] = value
+            case let .bool(value):
+                anyDict[key] = value
+            case let .int(value):
+                anyDict[key] = value
+            case let .double(value):
+                anyDict[key] = value
+            case let .stringArray(value):
+                anyDict[key] = value
+            case let .boolArray(value):
+                anyDict[key] = value
+            case let .intArray(value):
+                anyDict[key] = value
+            case let .doubleArray(value):
+                anyDict[key] = value
+            case let .set(value):
+                anyDict[key] = value // Assuming AttributeSet conforms to Any
+            }
+        }
```

Comment:
why this switchcase ? all the cases are doing the same 
anyDict[key] = value

Reply:
each case enter the relevant type to anyDict[key]
In Swift, enums can have associated values, which allow each case to store additional information. This is useful when you want to define a set of related values that can take different types of associated data.